sys/net/gnrc/transport_layer/tcp/gnrc_tcp.c

```diff
@@ -156,23 +156,23 @@ static int _gnrc_tcp_open(gnrc_tcp_tcb_t *tcb, const uint8_t *target_addr, uint1
     }
 
     /* Call FSM with Event: CALL_OPEN */
-    ret = _fsm(tcb, GNRC_TCP_FSM_EVENT_CALL_OPEN, NULL, NULL, 0);
+    ret = _fsm(tcb, FSM_EVENT_CALL_OPEN, NULL, NULL, 0);
     if (ret == -ENOMEM) {
         DEBUG("gnrc_tcp.c : gnrc_tcp_connect() : Out of receive buffers.\n");
     } else if(ret == -EADDRINUSE) {
```

Comment:
While you're at it, you may also fix these lines.
(RIOT coding conventions require the else in a new line.